framework/src/modules/chain/bft/finality_manager.js

```diff
@@ -58,7 +58,7 @@ class FinalityManager extends EventEmitter {
 		this.finalizedHeight = finalizedHeight;
 
 		// Height up to which blocks have pre-voted
-		this.prevotedConfirmedHeight = 0;
+		this.chainMaxHeightFinalized = 0;
```

Comment:
`this.prevotedConfirmedHeight` are different than `this.chainMaxHeightFinalized`

Reply:
It was meant to be `chainMaxHeightPrevoted`, will update.
This was because when we were discussing the bug, it was hard to communicate/reference the LIP